index.html

```diff
@@ -10930,11 +10930,52 @@ <h2>A project for first-time contributions</h2>
                 </div>
                 <!-- ________ Velismo's card END ________  -->
 
+                <!-- ________ Matthew's card START ________  -->
+                <div class="card">
+                    <p class="name">Name</p>
```

Comment:
I have already merged your PR. Good contribution. But I realised that you forgot to put your name here. You can submit a new PR with this correction to have your name on the card